src/posts/2023-05-30-tietoevry/index.md

```diff
@@ -2,7 +2,7 @@
 date: '2023-05-30T15:56:00.000Z'
 title: 'Better policies, less tool sprawl: Tietoevry uses Cilium and Hubble'
 ogImage: ogimage.webp
-ogSummary: 'Learn about how Tietoevry uses Isovalent Enterprise for Cilium with Hubble to have advanced network policies'
+ogSummary: 'Learn how Tietoevry uses Hubble for observability and Isovalent Enterprise for Cilium to implement advanced network policies'
```

Comment:
Hubble is a part of Isovalent Enterprise for Cilium, this makes it seem like they are separate things